src/plugins/filedelete/filedelete.h

```diff
@@ -117,6 +118,13 @@ using handle_t = uint64_t;
 using handled_t = bool;
 using file_name_t = std::string;
 
+enum file_extraction_reason
+{
+    FILEEXTR_WRITE,
+    FILEEXTR_DELETE,
+};
+using file_extraction_reason_t = file_extraction_reason;
```

Comment:
It seems to me this enum could be be moved to private.h

Reply:
@tklengyel I can't do it because of Line 159:
```c++
std::map<std::pair<vmi_pid_t, handle_t>, std::pair<file_name_t, file_extraction_reason_t>> files;
```